### PR TITLE
fix(VField): use disabled as placeholder color

### PR DESCRIPTION
**Pull request by jd-solanki**
## Description

It will fix the dark color of placeholder for input & textarea compared to current version where ther's no color for placeholder. Moreover, we added placeholder color in [v2](https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/components/VInput/VInput.sass#L11-L13)

Screenshot 2022-08-04 at 12 50 44 PM

Without this changes - placeholder are darker than label
Screenshot 2022-08-04 at 12 51 09 PM

## Motivation and Context

As above

## How Has This Been Tested?

unit & visually

## Markup:

<details>

```vue
<script setup lang="ts">
  import { useTheme } from 'vuetify'
  const vuetifyTheme = useTheme()
  const toggleTheme = () => {
    vuetifyTheme.global.name.value = vuetifyTheme.global.name.value === 'dark' ? 'light' : 'dark'
  }
</script>

<template>
  <v-container>
    <v-btn color="primary" class="mb-4" @click="toggleTheme">Toggle Dark mode</v-btn>

    <div class="d-flex flex-column">
      <p>Some body text</p>
      <v-text-field placeholder="Placeholder" variant="outlined" />
      <v-text-field label="Label" variant="outlined" />

      <v-textarea placeholder="Placeholder" variant="outlined" />
      <v-textarea label="Label" variant="outlined" />
    </div>
  </v-container>
</template>
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by jd-solanki** [comment]
use this config for proper color as shown my screenshot:

<details>

```js
import '@mdi/font/css/materialdesignicons.css'
import { createVuetify } from 'vuetify/src/entry-bundler'
import { fa } from 'vuetify/src/iconsets/fa-svg'
import { aliases, mdi } from 'vuetify/src/iconsets/mdi'
import { ar, en, ja, sv } from 'vuetify/src/locale'

export default createVuetify({
  locale: {
    messages: {
      en,
      ar,
      sv,
      ja,
    },
  },
  icons: {
    defaultSet: 'mdi',
    aliases,
    sets: {
      mdi,
      fa,
    },
  },
  theme: {
    defaultTheme: 'light',
    themes: {
      light: {
        dark: false,
        colors: {
          primary: '#a169ff',
          secondary: '#677492',
          'on-secondary': '#fff',
          success: '#40c057',
          info: '#2f8be6',
          warning: '#f77e17',
          error: '#f55252',
          'on-background': '#5e5669',
          'on-surface': '#5e5669',
          'grey-50': '#FAFAFA',
          'grey-100': '#F5F5F5',
          'grey-200': '#EEEEEE',
          'grey-300': '#E0E0E0',
          'grey-400': '#BDBDBD',
          'grey-500': '#9E9E9E',
          'grey-600': '#757575',
          'grey-700': '#616161',
          'grey-800': '#424242',
          'grey-900': '#212121',
        },
        variables: {
          'medium-emphasis-opacity': 0.68,
        },
      },
      dark: {
        dark: true,
        colors: {
          primary: '#a169ff',
          secondary: '#677492',
          'on-secondary': '#fff',
          success: '#40c057',
          info: '#2f8be6',
          warning: '#f77e17',
          error: '#f55252',
          'on-background': '#E8E8F1',
          'on-surface': '#E8E8F1',
          'grey-50': '#2A2E42',
          'grey-100': '#2F3349',
          'grey-200': '#4A5072',
          'grey-300': '#5E6692',
          'grey-400': '#7983BB',
          'grey-500': '#8692D0',
          'grey-600': '#AAB3DE',
          'grey-700': '#B6BEE3',
          'grey-800': '#CFD3EC',
          'grey-900': '#E7E9F6',
        },
        variables: {
          'medium-emphasis-opacity': 0.68,
        },
      },
    },
  },
})
```

</details>

File: packages/vuetify/src/components/VField/VField.sass [hunk 1]
```diff
@@ -103,6 +103,13 @@
   input
     letter-spacing: inherit
 
+  $v-field-input: &
+  @at-root
+    #{$v-field-input} input,
+    textarea#{$v-field-input}
+      &::placeholder
+        opacity: var(--v-disabled-opacity)
```
Comment (KaelWD) [inline packages/vuetify/src/components/VField/VField.sass:111]:
```suggestion
  @at-root
    & input::placeholder,
    textarea#{&}::placeholder
      opacity: var(--v-disabled-opacity)
```

**Comment by jd-solanki** [comment]
Thanks for the suggestions @KaelWD 

Commited!

**Comment by KaelWD** [comment]
This seems too light, v2 used black/white with that opacity instead of the browser default `#757575`

**Comment by jd-solanki** [comment]
Hi,

Yes, I checked again and I found that it wasn't using the on-surface. I updated the PR again.

Moreover, according to my POV placeholder shouldn't get an error or success color based on the input state. The label is a different thing than a placeholder.
Screenshot 2022-08-04 at 4 13 24 PM

**Comment by KaelWD** [comment]
```
<template>
  <v-app-bar color="primary">
    <v-text-field variant="outlined" placeholder="placeholder" hide-details />
    <v-text-field variant="outlined" label="label" hide-details />
  </v-app-bar>
</template>
```

**Comment by jd-solanki** [comment]
Do you prefer using `currentColor`?

```sass
  @at-root
    & input::placeholder,
    textarea#{&}::placeholder
      color: currentColor
      opacity: var(--v-disabled-opacity)
```

Screenshot 2022-08-04 at 4 39 27 PM

**Comment by KaelWD** [comment]
Yeah that seems to work better unless I've missed something.

**Comment by jd-solanki** [comment]
Here's the latest change then.

**Review by KaelWD — APPROVED** [review]

**Pull request merged**